frontend/src/components/views/user/list/UserListView.tsx

```diff
@@ -78,6 +87,11 @@ export function UserListView (props: UserListViewProps) {
             USER_LIST_VIEW_CLASS_NAME
             + (className? ` ${className}` : '')
         }>
+            {
+            props.children ?
+                <AppModalContainer t={t} modal={AppModalType.EDIT_USER_MODAL} />
+                : null
+                }
```

Comment:
This logic seems very odd. 

The `AppModalContainer` should be enabled in `DashboardApp`, not specific view components, and it already is there.